assets/src/scss/pages/_resistance-hub-campaign.scss

```diff
@@ -17,7 +17,7 @@
 @mixin btn-cta {
   position: static;
   width: 100%;
-  font-size: 25px;
+  font-size: 18px;
   line-height: 100%;
   padding-top: 18px;
```

Comment:
If we can use spacing variables could be great and standarize the spacing values with the rest of our code. I know that initially we didn't implement that but the projects is now growing more.